test/dialing/direct.node.js

```diff
@@ -171,66 +173,90 @@ describe('Dialing (direct, TCP)', () => {
     expect(dialer.queue.pending).to.equal(0)
     expect(dialer.queue.size).to.equal(0)
   })
-})
-
-describe('libp2p.dialer', () => {
-  let peerInfo
-  let remotePeerInfo
-  let libp2p
-  let remoteLibp2p
-  let remoteAddr
-
-  before(async () => {
-    const [peerId, remotePeerId] = await Promise.all([
-      PeerId.createFromJSON(Peers[0]),
-      PeerId.createFromJSON(Peers[1])
-    ])
-
-    peerInfo = new PeerInfo(peerId)
-    remotePeerInfo = new PeerInfo(remotePeerId)
 
-    remoteLibp2p = new Libp2p({
-      peerInfo: remotePeerInfo,
-      modules: {
-        transport: [Transport],
-        streamMuxer: [Muxer],
-        connEncryption: [mockCrypto]
-      }
+  describe('libp2p.dialer', () => {
+    let peerInfo
+    let remotePeerInfo
+    let libp2p
+    let remoteLibp2p
+    let remoteAddr
+
+    before(async () => {
+      const [peerId, remotePeerId] = await Promise.all([
```

Comment:
Should we have an util `createPeerInfoFromFixtures`? We could provide `n` and receive `n` `PeerInfos`. This is being replicated in a lot of tests

Reply:
I created this on [libp2p/js-libp2p#470](https://github.com/libp2p/js-libp2p/pull/470)

Reply:
Okay, once we get through these next few merges I can go through and do some deduplication.